### ISSUE TITLE
Misleading TODO item for "contains" entry in test YAML file.

### ISSUE THREAD

**Issue by charles-plessy** [nf-core/tools#1082]
## Description of the bug

`nf-core modules create-test-yml` sometimes inserts the following TODO item:

```yaml
# TODO nf-core: file md5sum was variable, please replace this text with a string found in the file instead
```

However, following the instructions results in a borken file, as the `contains` map need to contain an array. 

## Expected behaviour

How about either:

 - edit the `TODO` so that it asks the users to `...replace this text with an array of strings found...`
 - or insert the `TODO` in an array so that replacing the text results in the correct YAML structure, like:
     ```yaml
       files:
         - path: path/to/file
            contains:
              - "# TODO nf-core: file md5sum was variable, please replace..."
     ```

**Comment by ewels** [comment]
Relevant code: https://github.com/nf-core/tools/blob/d49bf614c431c092baa57839b96de92dd7ceaa67/nf_core/modules/test_yml_builder.py#L238-L244

Should be a simple enough change - just wrapping an extra set of `[]` around the string 👍🏻 

Do you fancy putting together a PR @charles-plessy?

**Comment by KevinMenden** [comment]
closed by #1089
